content/6_Advanced/Multiplicative.mdx

```diff
@@ -218,19 +218,240 @@ identity function $I$. Sometimes, a more careful observation is needed.
 <CPPSection>
 
 ```cpp
+#include <bits/stdc++.h>
+using namespace std;
+using ll = long long;
+
+// BeginCodeSnip{ModInt}
```

Comment:
where's this from

Reply:
It's the one I use, which belongs to tourist

Reply:
surely there's a shorter one that suffices for this code 💀

Reply:
or wait does the old code just implicitly use this

Reply:
The previous author of the code was using such a template but it was not mentioned there. So I just added what I use often.